core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -50,6 +53,17 @@
  */
 public abstract class ServerCall<RespT> {
   /**
+   * {@link Attributes.Key} for the remote address of server call.
+   */
+  public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+          Attributes.Key.of("io.grpc.RemoteAddr");
```

Comment:
Since it's now in a server-specific location, maybe the string should be `io.grpc.ServerCall.RemoteAddr", so that we can add the client-side equivalent when we need to? Ditto to the other key.